src/edit/DistortableImage.Edit.js

```diff
@@ -121,7 +129,13 @@ L.DistortableImage.Edit = L.Handler.extend({
 		L.DomEvent.off(window, 'keydown', this._onKeyDown, this);
 
 		overlay.fire('deselect');
-  },
+	},
+	
+	_initToolbar: function() {
+		this._showToolbar();
+    this.toolbar._hide();
+    this.toolbar._tip.style.opacity = 0; 
```

Comment:
Curious about this particular functionality. Can you explain this a bit? Thanks!